packages/node/example/multiSession/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "@inrupt/demo-authn-node-script",
+  "version": "1.0.0",
+  "description": "Demo for session management in NodeJS",
+  "main": "src/index.js",
+  "scripts": {
+    "build": "tsc -p tsconfig.json"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/inrupt/solid-client-authn-js.git"
+  },
+  "author": "Inrupt, Inc.",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/inrupt/solid-client-authn-js/issues"
+  },
+  "homepage": "https://github.com/inrupt/solid-client-authn-js#readme",
+  "dependencies": {
+    "cookie-session": "^1.4.0",
+    "express": "^4.17.1"
+  },
```

Comment:
Just curious - but is this really the intent here - i.e., to not depend on SCAN at all, but instead import directly from the 'dist' code using `import x from '../../dist/y';`...? I guess that makes it easy to test code changes locally, but slightly awkward to lift this demo code and re-use it for my own NodeJS server...
Would it warrant a mention in the README - i.e., "This demo imports code from SCAN directly rather than depending on a deployed npm module. This allows you to..., but means..." ...?

Reply:
Yeah, directly importing the code from the `dist` folder makes it much easier to use with local changes, without needing to publsh the package on verdaccio or NPM.But you are right, that means one cannot copy-paste and run this code in an environment where `solid-client-authn-node` isn't available. i'll add a comment directly in the code in the imports section to specify this.

Reply:
9a6dec5215e71f56d4b543a238051cb677b0a823